.github/workflows/api.yaml

```
@@ -42,8 +54,20 @@ jobs:
         run: |
           echo "GALAXY_CONFIG_OVERRIDE_METADATA_STRATEGY=extended" >> $GITHUB_ENV
           echo "GALAXY_CONFIG_OVERRIDE_OUTPUTS_TO_WORKING_DIRECTORY=true" >> $GITHUB_ENV
+      - if: github.event_name == 'workflow_dispatch'
+        run: |
+          echo "$COMMAND_ENV" | tr " " "\n" >> $GITHUB_ENV
+        env:
+          COMMAND_ENV: ${{ github.event.inputs.env_vars }}
```

Would the simpler

```suggestion
          echo "${{ github.event.inputs.env_vars }}" | tr " " "\n" >> $GITHUB_ENV
```

work?

---

Yep, that works